src/api/integrations/chatbot/openai/services/openai.service.ts

```
@@ -140,716 +343,469 @@ export class OpenaiService {
+      },
+    });
 
-  private isJSON(str: string): boolean {
-    try {
-      JSON.parse(str);
-      return true;
-    } catch (e) {
-      return false;
+    let conversationHistory = [];
+
+    if (session && session.context) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)